### ISSUE TITLE
draft / autosave behavior

### ISSUE THREAD

**Issue by j3ll3yfi5h** [statamic/ideas#890]
- as a (entry) publish form is opened (also in stack mode), a temporary version could be generated.
- the temporary version should already have a UUID

This would potentially fix/solve following issues:
- https://github.com/statamic/cms/issues/5500
- https://github.com/statamic/ideas/issues/725#issuecomment-1282366048

https://discord.com/channels/489818810157891584/489825364902805505/917402516134690816

**Comment by jasonvarga** [comment]
Yup I'd love to rework it to work that way, that one day. It would be a significant change, though.
